runtime/service.go

```diff
@@ -627,6 +627,13 @@ func (s *service) StopVM(requestCtx context.Context, request *proto.StopVMReques
 		timeout = time.Duration(request.TimeoutSeconds) * time.Second
 	}
 
+	if info, err := s.machine.DescribeInstanceInfo(requestCtx); err != nil {
+		return nil, err
```

Comment:
How about wrap additional context with this error, such as errors.Wrap(err, "failed to get instance info")?

Reply:
You need to return the wrapped error. `errors.Wrap()` won't mutate the err :)

Reply:
@alakesh Please fix this one.